src/g_canvas.c

```diff
@@ -2019,9 +2039,9 @@ void g_canvas_setup(void)
         /* we prevent the user from typing "canvas" in an object box
         by sending 0 for a creator function. */
     canvas_class = class_new(gensym("canvas"), 0,
-        (t_method)canvas_free, sizeof(t_canvas),
-            CLASS_NOINLET | CLASS_MULTICHANNEL, 0);
-            /* here is the real creator function, invoked in patch files
+                             (t_method)canvas_free, sizeof(t_canvas),
+                             CLASS_NOINLET | CLASS_MULTICHANNEL, 0);
+        /* here is the real creator function, invoked in patch files
```

Comment:
the Pd-codebase indents comments by an extra 4 spaces (there's multiple occurences where you changed that comment-indentation for no obvious reason)

Reply:
I have missed this change, sorry. I will correct that in the next commit.